opensearch/src/test/java/org/opensearch/sql/opensearch/request/OpenSearchRequestBuilderTest.java

```diff
@@ -214,6 +222,69 @@ void testPushDownProject() {
         requestBuilder.getSourceBuilder());
   }
 
+  @Test
+  void testPushDownNested() {
```

Comment:
I think we need an IT and/or UI to prove that merging a query made by filter (`WHERE` clause -> `pushDown`) with a query made by `pushDownNested` works good.

Reply:
Thanks, see [8620525cb4e853db28597bbf19e2be5310434b3a](https://github.com/opensearch-project/sql/pull/1490/commits/8620525cb4e853db28597bbf19e2be5310434b3a) for the fix.